reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/scripting/ImportChecker.kt

```diff
@@ -31,17 +31,13 @@ class ImportChecker {
         )
 
         // Get all classes (each class is subtype of java.lang.Object)
+        // FIXME: logic should be revised here, as caught exception is too generic (what kind of exception was intended to catch?)
+        @Suppress("SwallowedException", "TooGenericExceptionCaught")
         reflections.getSubTypesOf(Object::class.java)
             // Only public classes can be imported
             .filter { Modifier.isPublic(it.modifiers) }
             .filter {
-                try {
-                    // Skip if no canonical name
-                    it.canonicalName != null
-                } catch (e: Throwable) {
-                    // Skip if canonical name cannot be resolved
-                    false
-                }
+                if (it == null) false else it.canonicalName != null
```

Comment:
Please double check it. I haven't found any possible thrown exceptions except an NPE here

Reply:
I answered below, this case is the same

Reply:
got it, I will revert it

Reply:
thank you a lot!

Reply:
fixed